.ci/run_tests.sh

```diff
@@ -29,56 +43,72 @@ else
     export TEST_CMD="bin/busted $BUSTED_ARGS,postgres,db"
 fi
 
-if [[ "$KONG_TEST_COVERAGE" = true ]]; then
-    export TEST_CMD="$TEST_CMD --keep-going"
-fi
-
 if [ "$TEST_SUITE" == "integration" ]; then
+    integration_test_file=/tmp/integration_tests.$$.txt
+    trap "rm -f $integration_test_file" 0
+    # We're sorting the list of tests in a random order based on the
+    # pseudo random number generator (PRNG) of bash.  The PRNG is
+    # seeded with 0 to ensure that it always creates the same "random"
+    # sequence, making the sort order stable across workflow runs.
+    # This should level the run time between the two splits better
+    # than performing a static split based on directory name.
+    # Previously, all 05-proxy tests were run in the second split,
+    # which took three times as long as running all the other tests.
+    find spec/02-integration -name '*_spec.lua' \
+        | (RANDOM=0; while read line ; do echo "$RANDOM $line" ; done) \
+        | sort -n \
+        | awk '{print $2}' > $integration_test_file
+    test_count=$(echo $(wc -l < $integration_test_file))
```

Comment:
nit: is the `echo` needed here?

```suggestion
    test_count=$(wc -l < $integration_test_file)
```

Reply:
The `echo` is needed to remove the leading space from the `test_count` variable, as that will trip later commands if present.